src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/updatesubmission/changelist/cessations/CorporateEntityBeneficialOwnerCessation.java

```diff
@@ -0,0 +1,27 @@
+package uk.gov.companieshouse.overseasentitiesapi.model.updatesubmission.changelist.cessations;
+
+import java.util.Date;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class CorporateEntityBeneficialOwnerCessation extends BeneficialOwnerCessation {
+
+    @JsonProperty("corporateName")
+    private String corporateName;
+
+    public CorporateEntityBeneficialOwnerCessation(String appointmentId,
+            String appointmentType,
```

Comment:
Don't need to have appointmentType in constructor arg list. Can set to 5008.

Reply:
fixed